spec/features/bus_delete_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+describe 'deleting a bus - admin' do 
+	before :each do 
+		create :bus
+		when_current_user_is :admin
+		visit buses_url
+	end
+	context 'with admin privelege' do 
+		it 'delete a bus' do 
+			expect(page).to have_text 'Bus 1'
```

Comment:
Better to bind the bus to some variable, e.g.:

```ruby
let!(:bus) { create :bus }
```

and then dynamically refer to its name later:

```ruby
expect(page).to have_text bus.number
```

so that your tests won't fail if we change the factory later.